### PR TITLE
Kati user widget

### PR DESCRIPTION
**Pull request by andrzej-onja**

### CODE REVIEW

File: src/components/HeaderNavLink/List.tsx [hunk 1]
```diff
@@ -2,15 +2,17 @@ import React, { FC } from 'react'
 import styled from 'styled-components'
 
 export const ParentList: FC = ({ children }) => {
+  console.log(children);
```
Comment (andrzej-onja) [inline src/components/HeaderNavLink/List.tsx:5]: should be no console left

File: src/components/HeaderNavLink/HeaderNavLink.tsx [hunk 2]
```diff
@@ -50,19 +51,52 @@ export interface LinkTypes {
...
+//   } else {
+//     <p>This is NOT my account</p>
+//   }
+// });
+
+const profileName = loggedIn.map(item =>(
+    <p>{item.text}</p>
+))
+
+console.log(profileName);
```
Comment (ganamavo) [inline src/components/HeaderNavLink/HeaderNavLink.tsx:72]: We should remove this console from here

File: src/components/HeaderNavLink/HeaderNavLink.tsx [hunk 3]
```diff
@@ -50,19 +51,52 @@ export interface LinkTypes {
...
+
 export const HeaderNavLink: FC<ItemType> = ({ path, text, imgSrc, alt }) => (
+  
   <Item>
-    <Link to={`/${path}`} data-testid={text}>
+    <Link to={`/${path === "My Account"
+        ? 
+        <Dropdown>
+          <Dropbtn>{profileName}</Dropbtn>
+          <Dropdowncontent>
```
Comment (ganamavo) [inline src/components/HeaderNavLink/HeaderNavLink.tsx:83]: I'd change the component name into `DropdownContent` as it is the convetional naming

File: src/components/HeaderNavLink/HeaderNavLink.tsx [hunk 4]
```diff
@@ -89,3 +118,44 @@ const Item = styled.li`
...
+const Dropdown = styled.div ` 
+  position: relative;
+  display: inline-block;
+  :hover {
+    display: block;
+    background-color: #3e8e41;
+  }
+`
+
+const Dropdowncontent = styled.div `
```
Comment (ganamavo) [inline src/components/HeaderNavLink/HeaderNavLink.tsx:142]: It's here actually

File: src/pages/MyAccount.tsx [hunk 5]
```diff
@@ -4,11 +4,13 @@ import MyAccount from '../components/MyAccount/MyAccount'
 import { adsSelector } from '../redux/slices/adsSlice'
 
 export function MyAccountPage() {
-  const offers = useSelector(adsSelector)
+  const offers = useSelector(adsSelector);
```
Comment (ganamavo) [inline src/pages/MyAccount.tsx:7]: Is the use the of `useSelector()`  here ok?

**Comment by andrzej-onja** [comment]
closing as it was a part of graded project only

**Pull request closed without merging**